sympy/integrals/laplace.py

```diff
@@ -1502,6 +1507,26 @@ def _inverse_laplace_apply_simple_rules(f, s, t):
     return None
 
 
+def _inverse_laplace_diff(f, s, t, plane):
+    """
+    Helper function for the class InverseLaplaceTransform.
+    """
+    a = Wild('a', exclude=[s])
+    n = Wild('n', exclude=[s])
+    g = WildFunction('g')
+    ma = f.match(a*Derivative(g, (s, n)))
+    if ma and ma[n].is_integer:
+        m = [z.has(s) for z in ma[g].args]
+        if sum(m) == 1:
```

Comment:
Why does this condition `sum(m)` need to be checked?

We are checking `ma[g].args` but why does that matter?

What if I had something like
```
In [10]: s = symbols('s')

In [11]: Derivative(s*exp(s), s)
Out[11]: 
d ⎛   s⎞
──⎝s⋅ℯ ⎠
ds 
```

Reply:
The `g` is a `WildFunction` and can only match functions, so `Derivative(s*exp(s), s)` will not be matched by this function.

The purpose of this is that if the function that is matched has multiple arguments, like `besselj( , )`, then `s` should be present in exaxtly one of them.

I don't now whether this restriction is absolutely necessary, maybe it is too restrictive, but my mathematical knowledge does not go far enough to be certain that this rule would be generally valid for multiple-argument functions that contain `s` in several arguments.

Reply:
> my mathematical knowledge does not go far enough to be certain that this rule would be generally valid for multiple-argument functions that contain `s` in several arguments.

Mathematically any expression that involves `s` can be thought of as just a function of `s` having one argument if you just define a new function with a new name:
```python
f(s) := besselj(s, s)
g(s) := s*exp(s)
```

Reply:
So I was overthinking things here. I made a few checks, not with `exp`, because that is a time shift, but with stuff like `s*F(s)`, and removing the check causes no problems whatsoever.